cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/curator-manager/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: curator-manager
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: curator-report-manager
```

Comment:
the directory is called `curator-manager` while the name here is `curator-report-manager`. They should match.

If you do change the directory name, then also update the kustomization file.

Reply:
Metadata name and directory name are same only curator-manager. That’s a cluster role curator-report-manager.

Reply:
thanks, my bad.